governance/program/tests/process_create_realm.rs

```diff
@@ -1,4 +1,4 @@
-#![cfg(feature = "test-sbf")]
+// #![cfg(feature = "test-sbf")]
```

Comment:
pls explain or revert

Reply:
this is fixed in the versioned-transactions PR.